terragrunt/env/staging/env_vars.hcl

```diff
@@ -0,0 +1,9 @@
+inputs = {
```

Comment:
Can the `staging` env be removed?

Reply:
Removed it!

Reply:
Sorry - totally my misunderstanding. Whole file deleted. And it totally makes more sense since we do have only 1 environment.